backend/src/main/java/mpd/MpdServiceImpl.java

```diff
@@ -39,7 +43,7 @@ public MpdServiceImpl(String server, int port) {
     public void isConnected() {
```

Comment:
The naming is a little bit confusing, since the `is`-prefix implies to return a boolean instead of an Exception being thrown. Maybe we want to rename it to `ensureConnection()` or just let it return a boolean and let the caller do the error handling?

Reply:
Or is it even enough to do this connection check inside the `configure` method?

Reply:
True that! I renamed the function to ensureConnection(). Good comment!